src/annotations/AnnotationFile.php

```diff
@@ -69,6 +75,25 @@ public function __construct($path, array $data)
         $this->data = $data;
         $this->namespace = $data['#namespace'];
         $this->uses = $data['#uses'];
+
+        if (isset($data['#losingMethods'])) {
+            foreach ($data['#losingMethods'] as $class => $methods) {
+                $this->losingMethods[$class] = array_map(array($this, 'resolveMethod'), $methods);
+            }
+        }
+    }
+
+    /**
+     * Qualify the class name in a method reference like 'Class::method'.
+     *
+     * @param string $raw_method Raw method string.
+     *
+     * @return string
+     */
+    public function resolveMethod($raw_method)
+    {
+        list($class, $method) = explode('::', $raw_method);
```

Comment:
Better to specify `2` as 3rd argument to `explode` call to clearly indicate that we expect 2 pieces.